github/ci/prow-deploy/tasks/deploy.yaml

```diff
@@ -1,3 +1,32 @@
+# FIXME: use an image with this binary installed
+- name: install kustomize
```

Comment:
This works. Another approach could be to create a temp dir with tempfile module, and then removing it at the end. 
Pro:
 - cleaner /tmp 
 - Makes update of binaries possible (by default get_url force is off, so if the file exists, it will not be overwritten)
Cons:
- Additional complexity
- installation would not be reusable manually after playbook completes

Reply:
Yep, i also thought about the tempfile module but as I see it this would work only inside ansible roles, when we call render-environment-configs.sh we would need to pass somehow the location of the yq binary. I think we can go ahead with the simple /tmp location and rethink the approach later.